turbinia/api/cli/README.rst

```diff
@@ -0,0 +1,209 @@
+
+Usage
+^^^^^
+
+You can view the help menu with the ``-h`` argument
+
+.. code-block::
+
+   $ turbinia-client -h
+
+   Usage: turbinia-client [OPTIONS] COMMAND [ARGS]...
+
+     Turbinia API command-line tool (turbinia-client).
+
+                             ***    ***
+                              *          *
+                         ***             ******
+                        *                      *
+                        **      *   *  **     ,*
+                          *******  * ********
+                                 *  * *
+                                 *  * *
+                                 %%%%%%
+                                 %%%%%%
+                        %%%%%%%%%%%%%%%       %%%%%%
+                  %%%%%%%%%%%%%%%%%%%%%      %%%%%%%
+     %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%  ** *******
+     %%                                                   %%  ***************
+     %%                                (%%%%%%%%%%%%%%%%%%%  *****  **
+       %%%%%        %%%%%%%%%%%%%%%
+       %%%%%%%%%%                     %%          **             ***
+          %%%                         %%  %%             %%%           %%%%,
+          %%%      %%%   %%%   %%%%%  %%%   %%%   %%  %%%   %%%  %%%       (%%
+          %%%      %%%   %%%  %%%     %%     %%/  %%  %%%   %%%  %%%  %%%%%%%%
+          %%%      %%%   %%%  %%%     %%%   %%%   %%  %%%   %%%  %%% %%%   %%%
+          %%%        %%%%%    %%%       %%%%%     %%  %%%    %%  %%%   %%%%%
+
+     This command-line tool interacts with Turbinia's API server.
+
+     You can specify the API server location in ~/.turbinia_api_config.json
+
+   Options:
+     -c, --config_instance TEXT  A Turbinia instance configuration name.
+                                 [default: (dynamic)]
+     -p, --config_path TEXT      Path to the .turbinia_api_config.json file..
+                                 [default: (dynamic)]
+     -h, --help                  Show this message and exit.
+
+   Commands:
+     config  Get Turbinia configuration.
+     jobs    Get a list of enabled Turbinia jobs.
+     result  Get Turbinia request or task results.
+     status  Get Turbinia request or task status.
+     submit  Submit new requests to the Turbinia API server.
+
+Getting the server configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block::
+
+   turbinia-client config list
+
+Getting request or task information
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To get the status of a Turbinia request:
+
+.. code-block::
+
+   turbinia-client status request <request_id>
+
+For task status:
+
+.. code-block::
+
+   turbinia-client status task <task_id>
+
+where :raw-html-m2r:`<request_id>` and :raw-html-m2r:`<task_id>` are the respective Turbinia request or task identifiers.
```

Comment:
Looks like this isn't formatting quite correctly, or at least the github pages don't render it correctly.  Is that just specific to github, but would be just fine on readthedocs.io?

Reply:
Ahh good catch, this is due to me changing the role name and not adjusting all references. Will push a fix commit!